lms/djangoapps/course_home_api/course_metadata/v1/tests/test_views.py

```diff
@@ -18,10 +18,9 @@ class CourseHomeMetadataTests(BaseCourseHomeTests):
     """
     Tests for the Course Home Course Metadata API
     """
-    @classmethod
-    def setUpClass(cls):
-        BaseCourseHomeTests.setUpClass()
-        cls.url = reverse('course-home-course-metadata', args=[cls.course.id])
+    def setUp(self):
+        super().setUp()
+        self.url = reverse('course-home-course-metadata', args=[self.course.id])
```

Comment:
This and similar cleanup in this PR is due to me wanting to add a test that modifies the course in the modulestore. So that means we have to use `ModuleStoreTestCase` instead of `SharedModuleStoreTestCase` and make courses in `setUp` not `classSetUp`.

I figured this might not be such a unique desire, so chose to edit the base class and have everyone use the same code path, rather than making my test be a special butterfly. But I'm open to arguments that I should have left this all alone.